docs/current_docs/api/clients-sdk.mdx

```diff
@@ -199,6 +233,10 @@ the main object needs to be named `MyModule`.
 ```php file=./snippets/functions/functions-complex/php/src/MyModule.php
 ```
 </TabItem>
+<TabItem value="Java">
+```php file=./snippets/functions/functions-complex/java/src/main/java/io/dagger/modules/mymodule/MyModule.java
```

Comment:
```suggestion
```java file=./snippets/functions/functions-complex/java/src/main/java/io/dagger/modules/mymodule/MyModule.java
```

Reply:
Heads up that this code snippet is already part of https://github.com/dagger/dagger/pull/9648/files#diff-18c213121c56c8604bb3932971774db796b6f0f4fa156878c1acfedd60a875ef